.travis.yml

```diff
@@ -38,14 +38,14 @@ env:
     # sort modules by test time with quickest modules first
     - MODULE='server'
     - MODULE='hadoop-parent/janusgraph-hadoop-2'
+    - MODULE='es' ARGS='-Pelasticsearch6'
```

Comment:
I switch these to see a result early. I will revert this changes later.

Reply:
You can comment the other modules out if you're just testing the es builds so it's quicker and gentler on Travis. Also you might setup your own Travis job to monitor your fork so you're able to stop/restart jobs as needed.

Reply:
I canceled the other builds to so now the latest (with just es modules is running).